config/locales/models_and_forms.en.yml

```diff
@@ -118,10 +118,17 @@ en:
       activity:
         actual_end_date: Actual end date (optional)
         actual_start_date: Actual start date (optional)
+        dates: What are the start and end dates for this %{level}?
```

Comment:
I think the `helpers.fieldset.x` key is used by the govuk form builder gem to automatically match onto the HTML element `fieldset` without us needing to specify it in each view. I also think `fieldset` is only used for radio form fields like status, as well as dates, so these might be different elements.

I reckon we could add locale values which are then automatically picked up by the gem. If we place them in `helper.label` or `helper.fieldset` following [the gem documentation](https://govuk-form-builder.netlify.app/building-blocks/localisation/#customising-locale-structure) we could remove all the logic we're adding to the views here.

```
helpers:
  label:
    person:
      role: What role do you play?
```

Reply:
This list now looks similar to the `page_title` copy [over in en.yml](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/config/locales/en.yml#L347). Do we need to remove or change what's in that file?

What is the approach to our content now? Is it

- `en.page_title` is used only for the browser page title
- `helpers.x is used for form labels/titles
- `activerecord.attributes` are used for error messages

Reply:
I don't think we can use the formbuilder gem 'magic' when we are intopolating the string - but I agree that it would be good to settle on some kind of structure for all the keys.

Reply:
Ah, something else that is relevant: we recently make a change to the gem config to point it to our form labels in `activerecord.attributes` instead following Leekys earliy lead. https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/config/initializers/form_builder.rb#L10. This was intended so that the form labels and their error message would always match up https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/config/locales/models_and_forms.en.yml#L9.

Reply:
Yes I tried editing the `activerecord.attributes` labels but that f*cked up the error messages, so you'd get something like "*What are the start and end dates for this project? can't be blank*" on error. Hence adding the fieldset legends instead.

Reply:
> This list now looks similar to the page_title copy over in en.yml. Do we need to remove or change what's in that file?

Well no because Louise did that in the first version of this PR (which I closed) and it messed up the page titles 🤦